### ISSUE TITLE
Fails on first step of the stack with S3 error

### ISSUE THREAD

**Issue by JonSmart** [aws-quickstart/quickstart-tableau-server#108]
> S3 error: The bucket you are attempting to access must be addressed using the specified endpoint. Please send all future requests to this endpoint. For more information check http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html

>The following resource(s) failed to create: [Infrastructure].

I am attempting to create this within eu-west-2, however I believe it is having trouble accessing the infrastructure S3 resources with the default configuration.

I am hoping I have just missed a simple step.

**Comment by vsnyc** [comment]
@JonSmart - how did you deploy this, i.e. from launch links we publish at https://aws.amazon.com/quickstart/architecture/tableau-server/ or cloning this repo and running it from your own bucket? Also, can you share the parameters you used redacting the license key and passwords?

**Comment by JonSmart** [comment]
Launched from the links published, then switched to London region so that I could select the VPC.  I am not sure sharing the initial set of parameters makes any difference - I have since copied the initial cloudFront template and manually overidden the `TemplateURL` so that it isn't constructed using the default values - as it appears to run from eu-west it required the URL to be `https://aws-quickstart.s3.amazonaws.com/quickstart-tableau-server/templates/tableau-server-infrastructure.template.yaml` - Note the lack of region...
The template currently attempts to use
`https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}submodules/quickstart-aws-vpc/templates/aws-vpc.template.yaml`

Which is when populated with the default values becomes:

`https://aws-quickstart.s3.us-east-1.amazonaws.com/quickstart-tableau-server/templates/tableau-server-infrastructure.template.yaml`

Which I believe works publicly but doesn't appear to work from CloudFormation...

**Comment by vsnyc** [comment]
Thanks @JonSmart  - appreciate you looking into it. It was actually a typo that's caused this. The substitutions should have been such that the constructed URL in the default case becomes:

https://aws-quickstart-eu-west-2.s3.eu-west-2.amazonaws.com/quickstart-tableau-server/templates/tableau-server-existing-vpc.template.yaml

The TemplateURL for TableauStack is:
```
      TemplateURL:
        Fn::Sub:
          - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}templates/tableau-server-existing-vpc.template.yaml'
          - S3Region: !If [UsingDefaultBucket, !Ref 'AWS::Region', !Ref QSS3BucketRegion]
            S3Bucket: !If [UsingDefaultBucket, !Sub '${QSS3BucketName}-${QSS3BucketRegion}', !Ref QSS3BucketName]
```,

but it should be:

```
      TemplateURL:
        Fn::Sub:
          - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}templates/tableau-server-existing-vpc.template.yaml'
          - S3Region: !If [UsingDefaultBucket, !Ref 'AWS::Region', !Ref QSS3BucketRegion]
            S3Bucket: !If [UsingDefaultBucket, !Sub '${QSS3BucketName}-${AWS::Region}', !Ref QSS3BucketName]
```

I'll create a PR to fix this.

**Comment by vsnyc** [comment]
Fixed in #109
